api-messaging.md

```diff
@@ -0,0 +1,98 @@
+# Messaging API
+
+## Version
+
+- v1.0: Initial draft (2021-11-25)
+
+## General
+
+### Encoding
+
+UTF-8 encoding shall be used at all times.
+
+### Staging base URL
+
+https://api-staging.safereach.com/blaulicht
+
+### Live base URL
+
+https://api.safereach.com/blaulicht
+
+### Usage
+
+The API is a simple transactional messaging API.
+
+API requests are limited by the Fair Use Policy and configured account limits.
+
+### Authentication
+
+For API usage your `customerId`, an automatic alarm trigger user `username` and `password` are needed.
+
+### Models
+
+#### MessageTarget
+
+- channel[]: List of `SMS`, `EMAIL`, `VOICE`, `SAFEREACH`
+- msisdn: target phone number (mandatory for `SMS` and `VOICE`)
+- email: target email address (mandatory for `EMAIL`)
+
+> `SAFEREACH` will automatically determine the best message channel which will be the app in most cases
+
+**Example:**
+
+```json
+{
+  "channel": [
+    "SMS",
+    "VOICE",
+    "EMAIL"
+  ],
+  "msisdn": "+43123456789",
+  "email": "target@acme.com"
+}
+```
+
+## Send message
+
+_**/api/messaging/v1/send**_
```

Comment:
As I know, we currently do not have any versioning in our apis - correct me if I'm wrong. Therefore I would suggest to keep our `urls` consistent and avoid putting the versioning into the `url`. Instead I would put the versioning into a custom header e.g. `X-Api-Version: v1`. I'm open to discuss this suggestion 😃

Reply:
Discussed with @pmig. We will use the url versioning approach!